include/dgl/runtime/ndarray.h

```diff
@@ -139,11 +139,14 @@ class NDArray {
   /*!
    * \brief Copy data content from another array.
    * \param other The source array to be copied from.
+   * \param stream The stream to perform the copy on.
    * \note The copy may happen asynchrously if it involves a GPU context.
    *       DGLSynchronize is necessary.
    */
-  inline void CopyFrom(DLTensor* other);
-  inline void CopyFrom(const NDArray& other);
+  inline void CopyFrom(DLTensor* other,
+      DGLStreamHandle stream = nullptr);
```

Comment:
how does this work for CPU?

Reply:
@zheng-da I expanded the comment for the stream parameter to:
```
   * \param stream The stream to perform the copy on if it involves a GPU
   *        context, otherwise this parameter is ignored.
```

in 286ec68 .